src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -44,7 +46,7 @@ public async Task SendVerifyEmailEmailAsync(string email, Guid userId, string to
             WebVaultUrl = _globalSettings.BaseServiceUri.VaultWithHash,
             SiteName = _globalSettings.SiteName
         };
-        await AddMessageContentAsync(message, "VerifyEmail", model);
+        await AddMessageContentAsync(message, "Auth.VerifyEmail", model);
```

Comment:
Nice.

When we start moving to mailers, this will be less of a problem because the mailer will live next to the templates again